train/train_params.py

```diff
@@ -105,4 +106,6 @@ class TrainParams:
 #   Relevant only when also passing --save through the args, otherwise will be
 #   overridden to 0.
 # early_stop_nsamples:
-#   When >0, the number of batches to train before early stopping the training.
\ No newline at end of file
+#   When >0, the number of batches to train before early stopping the training.
+# seed:
+#   Random seed for reproducibility
```

Comment:
write that it can be overwritten by main's args. also write that it is set randomly if None, and then stored with the randomly set value. state that the default value is None. do a run and verify that indeed this happens (i.e. that it sets a random value if None, that it stores it properly, and that you can then go get this value from the logs and use it to make an identical training run)